internal/sidekick/api/model.go

```diff
@@ -369,6 +369,48 @@ func (m *Method) IsSimple() bool {
 		m.OperationInfo == nil && m.DiscoveryLro == nil
 }
 
+// IsStandardMethod returns true if the method is a standard method.
+func (m *Method) IsStandardMethod() bool {
```

Comment:
Right now, it would be more appropriate to call this `IsGetResourceMethod()`.  I assume we will have more methods that are "standard"?

Also: the comment is not super useful.  Maybe say something about why the method exists:

```
// IsStandardMethod simplifies the writing of mustache templates
```

Though I would think `HasStandardGetInfo` is closer to what happens here.

Reply:
I'll fix the comment.

> I assume we will have more methods that are "standard" 

Yes, I'm seding PRs one by one (I have them lined up), but we have one similar for delete, create, etc. So, this `IsStandardMethod`  will cover more than just get.

If I change the Standard bit to AIPStandard, we will have: `AIPStandardGetInfo()`, `AIPStandardDeleteInfo()`, `AIPStandardCreateInfo()` ... and `IsAIPStandard()` that returns true if the method "has any of the AIP standard info". Does that sound better?

Reply:
> I'll fix the comment.
> 
> > I assume we will have more methods that are "standard"
> 
> Yes, I'm seding PRs one by one (I have them lined up), but we have one similar for delete, create, etc. So, this `IsStandardMethod` will cover more than just get.

Ack. Sending one by one SGTM.

> If I change the Standard bit to AIPStandard, we will have: `AIPStandardGetInfo()`, `AIPStandardDeleteInfo()`, `AIPStandardCreateInfo()` ... and `IsAIPStandard()` that returns true if the method "has any of the AIP standard info". Does that sound better?

Mentioning the AIP SGTM.  If it gets too annoying we can remove it.